### PR TITLE
add information how DelimitedFiles can be used

### LINKED ISSUE #2772: Question: Small Delimited file into DataFrame
Hi,
I have read the following:
https://dataframes.juliadata.org/stable/man/importing_and_exporting/
Is there an alternative way to get a small text file delimited with '\t' into a dataframe in the first run without waiting 12 seconds for CSV.File to compile? 
The text file is different everytime.
Thank you

### PR DESCRIPTION
**Pull request by bkamins**
Fixes #2772

### CODE REVIEW

**Review by quinnj — APPROVED** [review]

File: docs/src/man/importing_and_exporting.md [hunk 1]
```diff
@@ -31,3 +31,84 @@ online [CSV.jl documentation](https://juliadata.github.io/CSV.jl/stable/).
...
+```
+
+Observe that in our example:
+* `header` is a `Matrix` therefore we had to pass `vec(header)` to `DataFrame`
+  constructor;
+* we broadcasted `identity` function over the `iris_raw` data frame to perform
+  narrowing of `eltype` of columns of `iris_raw`; the reason is that read in by
+  the `readdlm` function is stored into a `data` `Matrix` so all columns in
+  `iris_raw` initially have the same `eltype` -- in this case it had to be `Any`
+  as some of the folumns are numeric and some are string.
```
Comment (nalimilan) [inline docs/src/man/importing_and_exporting.md:100]:
```suggestion
  as some of the columns are numeric and some are string.
```

File: docs/src/man/importing_and_exporting.md [hunk 2]
```diff
@@ -31,3 +31,84 @@ online [CSV.jl documentation](https://juliadata.github.io/CSV.jl/stable/).
 
 For reading and writing tabular data in Apache Arrow format use
 [Arrow.jl](https://github.com/JuliaData/Arrow.jl)
```
[the remark this answers is not in the document]
Comment (bkamins) [reply docs/src/man/importing_and_exporting.md:33]: OK.

[the remark this answers is not in the document]
Comment (bkamins) [reply docs/src/man/importing_and_exporting.md]: added

[the remark this answers is not in the document]
Comment (bkamins) [reply docs/src/man/importing_and_exporting.md]: while waiting for the approval I have added one more tutorial to the list of curated tutorials

[the remark this answers is not in the document]
Comment (bkamins) [reply docs/src/man/importing_and_exporting.md]: @nalimilan - looks good now?

File: docs/src/man/importing_and_exporting.md [hunk 3]
```diff
@@ -29,5 +31,95 @@ The behavior of CSV functions can be adapted via keyword arguments. For more
...
+## Other formats
+
+Other, selected, data formats are supported for reading and writing in the
+following packages:
+* Apache Arrow: [Arrow.jl](https://github.com/JuliaData/Arrow.jl)
+* Apache Avro: [Avro.jl](https://github.com/JuliaData/Avro.jl)
+* JSON: [JSONTables.jl](https://github.com/JuliaData/JSONTables.jl)
+* Parquet: [Parquet.jl](https://github.com/JuliaIO/Parquet.jl)
+* Stata, SPSS, and SAS: [StatFiles.jl](https://github.com/queryverse/StatFiles.jl)
+* XLSX: [XLSX.jl](https://github.com/felipenoris/XLSX.jl)
```
Comment (nalimilan) [inline docs/src/man/importing_and_exporting.md:125]:
```suggestion
* Microsoft Excel (XLSX): [XLSX.jl](https://github.com/felipenoris/XLSX.jl)
```

File: docs/src/man/importing_and_exporting.md [hunk 4]
```diff
@@ -1,5 +1,7 @@
 # Importing and Exporting Data (I/O)
 
+## CSV files
```
Comment (nalimilan) [inline docs/src/man/importing_and_exporting.md:3]:
```suggestion
## CSV Files
```

File: docs/src/man/importing_and_exporting.md [hunk 5]
```diff
@@ -29,5 +31,95 @@ The behavior of CSV functions can be adapted via keyword arguments. For more
...
+
+As you can see the code required to transform `iris` into a proper input to the
+`writedlm` function so that you can create the CSV file having the expected
+format is not easy. Therefore CSV.jl is the preferred package to write CSV files
+for data stored in data frames.
+
+## Other formats
+
+Other, selected, data formats are supported for reading and writing in the
+following packages:
```
Comment (nalimilan) [inline docs/src/man/importing_and_exporting.md:119]:
```suggestion
Other data formats are supported for reading and writing in the following packages
(non exhaustive list):
```

File: docs/src/man/importing_and_exporting.md [hunk 6]
```diff
@@ -29,5 +31,95 @@ The behavior of CSV functions can be adapted via keyword arguments. For more
...
+* Apache Arrow: [Arrow.jl](https://github.com/JuliaData/Arrow.jl)
```
Comment (nalimilan) [inline docs/src/man/importing_and_exporting.md:120]:
```suggestion
* Apache Arrow (including Feather v2): [Arrow.jl](https://github.com/JuliaData/Arrow.jl)
* Apache Feather (v1): [Feather.jl](https://github.com/JuliaData/Feather.jl)
```

**Review by nalimilan — APPROVED** [review]
Just a few suggestions.

**Comment by bkamins** [comment]
Thank you!

**Pull request merged**